lib/matplotlib/text.pyi

```diff
@@ -104,6 +105,8 @@ class Text(Artist):
     def set_parse_math(self, parse_math: bool) -> None: ...
     def get_parse_math(self) -> bool: ...
     def set_fontname(self, fontname: str | Iterable[str]): ...
+    def get_antialiased(self) -> bool | None: ...
```

Comment:
Can we arange things so the value can never be None?

Reply:
Done